platforms/documentation/docs/src/docs/userguide/best-practices/bp_general.adoc

```diff
@@ -15,6 +15,53 @@
 [[best_practices_general]]
 = General Gradle Best Practices
 
+[[use_latest_minor_versions]]
+== Use the Latest Minor Version of Gradle and Plugins
+
+Stay on the latest minor version of the major Gradle release you're using, and regularly update your plugins to the latest compatible versions.
+
+=== Explanation
+
+Each new Gradle minor release includes important bug fixes, performance improvements, and deprecation warnings that help you prepare for future breaking changes.
+
+Staying current with minor versions helps you:
+
+* Benefit from performance and stability improvements.
+* Catch deprecations early, while fixes are still simple.
+* Avoid large, painful upgrades caused by skipping too many versions.
```

Comment:
I think we might want to highlight that in our experience many smaller upgrades are easier than fewer big jump upgrades.